build-tools/packages/build-tools/README.md

```diff
@@ -214,3 +214,46 @@ Release group are basically group of packages managed by a workspace. `fluid-bui
 independent packages within the same repo. The repo structure is specified in `fluidBuild.config.cjs` at the root of
 the repo under `repoPackages` property. See [fluidBuild.config.cjs](../../../fluidBuild.config.cjs) for how it looks
 like.
+
+## Debug Traces
+
+`fluid-build` using the `debug` package to do traces for investigating and diagnosing problem. Here are some of the trace
+names
+
+### fluid-build:init
+
+Trace the initialization `fluid-build`, including root inference, package loading and selection (based on command line
+scopes).
+
+### fluid-build:task:definition
+
+To debug the logic that combined task and dependency definitions from the default in `fluidBuild.config.cjs` at the root
+of the
+repo and the package's `package.json`. It will dump the full combined definition for each package.
```

Comment:
```suggestion
Used to debug the logic that combines task and dependency definitions from the default in `fluidBuild.config.cjs` at the root
of the
repo and the local package's `package.json`. It will dump the full combined definition for each package.
```